rpcs3/Emu/System.h

```diff
@@ -261,7 +261,7 @@ struct cfg_root : cfg::node
 	{
 		node_core(cfg::node* _this) : cfg::node(_this, "Core") {}
 
-		cfg::_enum<ppu_decoder_type> ppu_decoder{this, "PPU Decoder", ppu_decoder_type::fast};
+		cfg::_enum<ppu_decoder_type> ppu_decoder{this, "PPU Decoder", ppu_decoder_type::llvm};
```

Comment:
Yes @scribam you can compile cmake with `-DWITHOUT_LLVM` or if you just don't have the correct LLVM version installed it will compile without LLVM.
With llvm set as default, you'll see LLVM blanked out, but it will be selected.